src/main/java/CreationShip/demo/DemoApplication.java

```diff
@@ -1,24 +1,21 @@
 package CreationShip.demo;
 
 import CreationShip.demo.IO.Acceptor;
-import CreationShip.demo.IO.AcceptorNIO;
+import CreationShip.demo.NIO.Handler;
 import CreationShip.demo.service.MessageService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.ApplicationContext;
-
-import java.util.concurrent.Executor;
 
 @SpringBootApplication
 public class DemoApplication implements CommandLineRunner{
```

Comment:
use h2 database to run it on developer machine.
Also maybe you need to add flyway/liquibase version system?